controllers/disruption_controller_test.go

```diff
@@ -315,22 +321,10 @@ var _ = Describe("Disruption Controller", func() {
 				Unsafemode: &chaosv1beta1.UnsafemodeSpec{
 					DisableAll: true,
 				},
-				Selector:   map[string]string{"foo": "bar"},
```

Comment:
If we're removing that because network disruption can't handle cgroups v2 right now, can we comment it out rather than removing it so we can add it back later please?

Reply:
It's more like an in-place substitution that doesn't affect what this test was originally testing (cleanup and gc), but uses a kind that we can test with v2 right now. Do we still need to change it back? I can add a comment

Reply:
Got it :D no need to change it back